### DISCUSSION TITLE
[Projects] Auto-add to project workflow seems not working for PRs changing from draft to "ready for review"

### DISCUSSION THREAD

**Discussion by RudyLu** [community/community#50561] [Projects and Issues]
### Select Topic Area

Bug

### Body

While trying out the new feature, auto-add to project workflow (announced with #44816), it works for general cases, like new PRs created.

While the title says "When the filter matches a new or updated item", but it seems not working for PRs when they have been changed from draft to "ready for review".
<img width="420" alt="Screenshot 2023-03-21 at 2 39 35 PM" src="https://user-images.githubusercontent.com/1704629/226534370-0d1960b9-f668-4b8e-8461-82ce71db2f7c.png">

Already found 2 occurrences of it.

Can someone confirm if this is expected and if yes, is it going to be addressed or any easy workaround?
Thanks in advance.

**Comment by david-wolgemuth** [comment]
We are also experiencing this with `is:pr is:merged` filter -- does not add pr to project

**Comment by vpavicic** [comment]
I have this example... 

I want a PR to be added to the project if it has/gets a certain label

![image](https://github.com/community/community/assets/95473291/189bb231-6317-4b2c-9daf-8b72627c5526)

It filters them out correctly, the message below changes to, for example, "See 6 existing items that match this query" and I always need to add them manually, nothing has ever been added automatically.

**Reply by darksciencebase** [reply]
I have the same problem with issues. So my filter looks like `is:issue label:"...label_name..."`. It filters them out correctly, the message below changes to "See X existing items that match this query", but the automation just does not work.

Amusingly, it does work for issues from another repository that get a specific label (I have duplicated the workflow).

**Comment by lukehinds** [comment]
It's still like this, seems workflows are no longer being maintained or improved by github.

[a comment by an automated account is not shown]

**Comment by RudyLu** [comment]
I’ve been feeling the friction of keeping up with PR reviews lately, so I decided to build a lightweight dashboard to streamline the process:
🔗 https://pr-dashboard.rudy.lu/

It’s a pure client-side tool—just plug in your GitHub PAT (saved locally in your browser), and you’re ready to go. 
No complex setup, just a clearer view of what needs your attention.

If you’re a dev or EM looking to speed up your review cycle, I’d love for you to give it a spin. 
Let me know what you think!
